### ISSUE TITLE
Convert to LaTeX with PSB template

### ISSUE THREAD

**Issue by trangdata** [lelaboratoire/rethink-prs-ms#7]
I have added a few lines to [build.sh](https://github.com/lelaboratoire/rethink-prs-ms/blob/e1ed878a88bac27584696af10642877e9a143da3/build/build.sh#L95-L112) following @slochower's guide to export to LaTeX file [here](https://github.com/manubot/rootstock/issues/249#issuecomment-509302510), but I'm not sure what the template (e.g. `nih4.tex`) should look like. I have tried the `.cls` file from provided by PSB but the build failed.

@dhimmel Any ideas?

PSB template zip file is attached.
[psb_template.zip](https://github.com/lelaboratoire/rethink-prs-ms/files/3447649/psb11x85_2e.zip)

**Comment by slochower** [comment]
I can send offline you the template that I used @trang1618, if you want. I basically combined an existing LaTeX template that I had around with the one that I generated via `pandoc -D latex`. I got right of a decent amount of `if`/`then`/`else` blocks in the default template, because I couldn't really tell what they were doing. It's not pretty.

**Comment by dhimmel** [comment]
We've been discussing LaTeX export in https://github.com/manubot/rootstock/issues/249, but this upcoming PSB deadline adds some urgency. I know @danich1 is also preparing a Manubot [manuscript](https://github.com/greenelab/text_mined_hetnet_manuscript) for PSB. So I will prioritize this.

> I can send offline you the template that I used @trang1618, if you want

@slochower is there some permission issue of this template that would prevent us from using it in production? Also Is a template necessary? I'm guessing the journal strips out most of the template content anyways?

**Comment by slochower** [comment]
@dhimmel No, not at all. It's just messy and probably not a good model of the template. I was also pressed by a deadline and so I didn't invest a lot of time getting things set optimally. Here is the template and an illustration of the formatting it provides. (You'll have to rename the template from TXT to TEX.) The naming is because I began with a LaTeX template that supposedly reproduces NIH formatting guidelines for R01 grants (although, no guarantees on that).

[nih4.txt](https://github.com/lelaboratoire/rethink-prs-ms/files/3452643/nih4.txt)

<img width="216" alt="nih4" src="https://user-images.githubusercontent.com/5393995/62225165-c5ed1680-b36c-11e9-9d5a-a395ea0ca6ea.png">

**Comment by dhimmel** [comment]
Noting the instructions at https://psb.stanford.edu/psb-online/psb-submit/:

> The accepted file format is PDF (Adobe Acrobat preferred). Attached files should be named with the last name of the first author (e.g. altman.pdf). Hardcopy submissions or unprocessed TEX or LATEX files or electronic submissions not submitted through the paper management system will be rejected without review.

> Submitted papers are limited to twelve (12) pages (not including the cover letter) in our publication format. The bibliography is included in the 12 page limit. Please format your paper according to the instructions below.
